lib/serverless-rag-on-aws-stack.ts

```diff
@@ -221,6 +224,20 @@ export class ServerlessRagOnAws extends Stack {
       autoDeploy: true,
     });
 
+    // Read the prompt-templates YAML file
```

Comment:
ah I think we need to have a default system prompt and one that can be overridden by the tenant... I think this might be lot of work and potentially should be done from the front-end